requirements.txt

```diff
@@ -4,7 +4,7 @@ requests-oauthlib==1.3.0
 requests-toolbelt==0.8.0
 pyjwt==1.6.4
 mock==2.0.0
-dpath==1.4.2
+dpath==2.0.1
```

Comment:
Do we have testing to ensure nothing breaks with this upgrade? I see the test below, but since that test is changing, I wanted to check if this was this used anywhere else in code that we need to be aware of.

Reply:
correct, only place it is used and it is tested.